### PR TITLE
Add TF EfficientNetB7

### PR DESCRIPTION
**Pull request by mariecwhite**
Also updates Tensorflow version to v2.13.0

### CODE REVIEW

File: .github/workflows/run_comparative_benchmark.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
     # Scheduled to run at 09:00 UTC and 21:00 UTC.
     - cron: '0 09,21 * * *'
   workflow_dispatch:
+  pull_request:
```
Comment (mariecwhite) [inline .github/workflows/run_comparative_benchmark.yml:16]: Reminder to remove.

File: common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    "efficientnetb3": tf.keras.applications.efficientnet.EfficientNetB3,
+    "efficientnetb4": tf.keras.applications.efficientnet.EfficientNetB4,
+    "efficientnetb5": tf.keras.applications.efficientnet.EfficientNetB5,
+    "efficientnetb6": tf.keras.applications.efficientnet.EfficientNetB6,
+    "efficientnetb7": tf.keras.applications.efficientnet.EfficientNetB7,
+}
+
+
+class EfficientNet(model_interfaces.InferenceModel, tf.Module):
+  """See https://huggingface.co/docs/transformers/model_doc/resnet for more information."""
```
Comment (pzread) [inline common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py:40]: The link seems wrong?

File: common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    self.batch_size = batch_size
+    self.model_name = model_name
+    self.input_size = MODEL_NAME_TO_INPUT_SIZE[model_name]
+    self.model = MODEL_NAME_TO_CLASS[model_name]()
+
+  def generate_default_inputs(self) -> Tuple[Any, ...]:
+    # TODO(#44): This should go away once we support different raw inputs.
+    image = utils.download_and_read_img(DEFAULT_IMAGE_URL,
+                                        width=self.input_size,
+                                        height=self.input_size)
```
Comment (pzread) [inline common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py:57]: I feel like we should resize the image in `preprocess` step. This way, we can use the same raw input image for models required different image sizes? (when we support non-default raw inputs)

**Review by pzread — APPROVED** [review]

File: common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py (hunk 2 above)
Comment (mariecwhite) [reply common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py:40]: Changed

File: common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py (hunk 3 above)
Comment (mariecwhite) [reply common_benchmark_suite/openxla/benchmark/models/tf/efficientnet/efficientnet_model.py]: That makes sense. I moved the resizing to the preprocessing.

**Pull request merged**